packages/tools/viewer/src/viewer.ts

```diff
@@ -981,6 +986,36 @@ export class Viewer implements IDisposable {
         await this._updateModel(undefined, undefined, abortSignal);
     }
 
+    /**
+     * Adds a 3D model from the specified URL to the scene.
+     * @remarks
+     * Will be stored in the viewer's list of loaded models.
+     * @param source A url or File or ArrayBufferView that points to the model to load.
+     * @param options The options to use when loading the model.
+     * @param abortSignal An optional signal that can be used to abort the loading process.
+     * @returns The loaded model.
+     */
+    protected async _addModel(source: string | File | ArrayBufferView, options?: LoadModelOptions, abortSignal?: AbortSignal): Promise<Model> {
+        const model = await this._loadModel(source, options, abortSignal);
+        this._loadedModelsBacking.push(model);
+        return model;
+    }

```

Comment:
I don't think we need this extra function, I think it is fine to just do `this._loadedModelsBacking.push(model)` directly from `_loadModel`.